src/meta.ml

```diff
@@ -1,16 +1,6 @@
 open Import
 
-type t =
```

Comment:
Why this change? I find it slightly nicer to have the main type first

Reply:
Just so that it's clear which  groups of types refer to each other. It's the 2nd commit, I can get rid of it.

Reply:
Ok, it's not a big deal I guess. But to be consistent `action` and `predicate` should be defined on their own.

Reply:
I just got rid of it, it's just a style change and I'm not sure it's applied consistently. I just thought you accidentally omitted it this one time.